controllers/users.js

```diff
@@ -221,7 +221,12 @@ const updateSelf = async (req, res) => {
       await userQuery.setIncompleteUserDetails(userId);
     }
 
-    const user = await userQuery.addOrUpdate(req.body, userId);
+    const data = {
```

Comment:
Store constants in respective files or create a utility function to get initial status.